docs/topics/wildfly-swarm/docs/reference/fractions/opentracing.adoc

```diff
@@ -0,0 +1,18 @@
+= OpenTracing
+
+
+.Maven Coordinates
+[source,xml]
+----
+<dependency>
+  <groupId>org.wildfly.swarm</groupId>
+  <artifactId>opentracing</artifactId>
+</dependency>
+----
+
+.Configuration
+
+swarm.opentracing.servlet.skipPattern:: 
+The servlet skip pattern as a Java compilable Pattern. Optional. Ex.: `/health-check`
```

Comment:
change 
> Ex.: `/health-check`

to 

> Example: `/health-check`

Reply:
Note to self: not easily fixable in our sources.